spec/features/collection_create_spec.rb

```diff
@@ -3,6 +3,7 @@
 RSpec.describe('Collection create', type: :request, integration: true) do
   let(:archivist1) { create :archivist1 }
   let(:archivist99) { create :archivist99 }
+  let(:wfs) { Dor::Config.workflow.client }
```

Comment:
This is the single biggest change between DS7 and DS8: the replacement of the workflows XML datastream with an external workflow service.